src/styles/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: var(--white-color);
+
+  --white-color: #fff;
+  --aqua-color: #00acdc;
+  --dark-blue-color: #060b35;
+  --light-grey-color: #616070;
```

Comment:
It's better to define variables in the [:root](https://developer.mozilla.org/en-US/docs/Web/CSS/:root) selector